.github/workflows/ci.yml

```
@@ -356,6 +356,7 @@ jobs:
       - name: Run specs
         run: |
           cd rabbit-hole
+          bash bin/ci/before_build.sh
           go test -v ./...
```

the path to `lavinmqctl` is wrong, and is `sudo` really required?
```suggestion
          go test -v ./...
        env:
          RABBITHOLE_LAVINMQCTL: ../bin/lavinmqctl
```